### ISSUE TITLE
Remove \fontspec from the examples

### ISSUE THREAD

**Issue by eg9** [Pomax/ucharclasses#7]
With the latest release of `fontspec`, an example such as

    \documentclass[12pt]{article}
    
    \usepackage{fontspec}
    \defaultfontfeatures{Mapping=tex-text}
    
    \usepackage[Latin, Thai]{ucharclasses}
    \setDefaultTransitions{\fontspec{Hoefler Text}}{}
    \setTransitionTo{Thai}{\fontspec[Scale=1]{Thonburi}}
    
    \begin{document}
    
    Test  ไทย
    
    \end{document}  

will make XeTeX into an endless loop (see [ucharclasses package does not work with TeX Live 2014's fontspec](http://tex.stackexchange.com/questions/188038/ucharclasses-package-does-not-work-with-tex-live-2014s-fontspec).

I believe it's advisable to declare fonts beforehand in any case and that the documentation should be accordingly modified; besides making the examples work, this has also the advantage of not inducing users into thinking that `\fontspec` should be used in the code.

**Comment by ickc** [comment]
I encountered the same problem as well.

One solution is this:

```latex
\documentclass{article} 

\usepackage[Latin, Greek, CJK, Hebrew]{ucharclasses}
\usepackage{fontspec}

\newfontfamily\greekfont{CMU Serif} %[Script=Greek]
\newfontfamily\hebrewfont{Narkisim} %[Script=Hebrew]
\newfontfamily\CJKfont{KaiTi} %[Script=CJK]

\setTransitionsForLatin{\normalfont}{}
\setTransitionsForGreek{\greekfont}{}
\setTransitionsFor{Hebrew}{\hebrewfont\setRTL}{\setLTR}
\setTransitionsForCJK{\CJKfont}{}

\usepackage{bidi}

\begin{document}

but my righteous one shall live by faith

ὁ δὲ δίκαιός μου ἐκ πίστεως ζήσεται

只是義人必因信得生。

בֶּאֱמוּנָת֥וֹ יִחְיֶֽה׃
      
\end{document}
```

**Comment by Pomax** [comment]
I've released 2.1 based on work in #14, and the documented use of fontspec in the updated `.pdf` doc should work quite well.
